docs/Manual/03 Schedules.md

```diff
@@ -2,7 +2,7 @@
 [//]: # (Version: v1.2.1)
 
 # Section 3: Schedules
-We begin this section with `nest` from [Section 2](<02%20Simple%20Affine%20Loop%20Nests.md>), which captures the logic of matrix-matrix multiplication. We use `nest` to create a `Schedule`, which controls the execution order of the nest iterations. Schedules are target-independent, in the sense that the same schedule can be used to emit code for multiple target platforms.
```

Comment:
Ah, the "We begin..." and "We use ..." are an intentional conversational voice used in writing the Manual. We should keep the "we" while fixing any grammatical issues found.

Something like:
We begin this section with `nest` from [Section 2](...) that captures .....  
We use `nest` to create a `Schedule` that ...

Reply:
We begin with nest from ...........
going with this one.